.ci/.e2e-tests-for-elastic-agent.yaml

```diff
@@ -4,10 +4,10 @@ SUITES:
     scenarios:
       - name: "Fleet"
         tags: "fleet_mode_agent"
-        platforms: ["centos8_arm64", "centos8_amd64", "debian_arm64", "debian_amd64", "sles15"]
+        platforms: ["centos8_arm64", "centos8_amd64", "debian_arm64", "debian_amd64", "sles15", "macos12"]
```

Comment:
Can we add these macos values to the `.e2e-tests.yaml` file too? It's described here https://github.com/elastic/e2e-testing/blob/main/.ci/README.md#jenkins-stages-and-platform-support

Reply:
Done.